src/lib/Component/EventSubscriber/RenderEventLegacyBridgeSubscriber.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @copyright Copyright (C) Ibexa AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+
+namespace Ibexa\AdminUi\Component\EventSubscriber;
+
+use Ibexa\AdminUi\Component\Event\RenderGroupEvent;
+use Ibexa\AdminUi\Component\Event\RenderSingleEvent;
+use Ibexa\AdminUi\Component\Registry;
+use Ibexa\Contracts\TwigComponents\Event\RenderGroupEvent as TwigComponentsRenderGroupEvent;
+use Ibexa\Contracts\TwigComponents\Event\RenderSingleEvent as TwigComponentsRenderSingleEvent;
+use Symfony\Component\EventDispatcher\EventDispatcherInterface;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+
+class RenderEventLegacyBridgeSubscriber implements EventSubscriberInterface
+{
+    protected Registry $registry;
+
+    protected EventDispatcherInterface $eventDispatcher;
+
+    public function __construct(
+        EventDispatcherInterface $eventDispatcher,
+        Registry $registry
+    ) {
+        $this->registry = $registry;
+        $this->eventDispatcher = $eventDispatcher;
+    }
+
+    public static function getSubscribedEvents(): array
+    {
+        return [
+            TwigComponentsRenderGroupEvent::class => ['onRenderGroup', 500],
+            TwigComponentsRenderSingleEvent::class => ['onRenderSingle', 500],
+        ];
+    }
+
+    public function onRenderGroup(TwigComponentsRenderGroupEvent $event): void
+    {
+        $this->eventDispatcher->dispatch(new RenderGroupEvent(
+            $this->registry,
+            $event->getGroupName(),
+            $event->getParameters(),
+        ), RenderGroupEvent::NAME);
+    }
+
+    public function onRenderSingle(TwigComponentsRenderSingleEvent $event): void
+    {
+        $this->eventDispatcher->dispatch(new RenderSingleEvent(
+            $this->registry,
+            $event->getGroupName(),
+            $event->getName(),
+            $event->getParameters(),
+        ), RenderSingleEvent::NAME);
+    }
```

Comment:
LGTM.

There is also an option to decorate `EventDispatcher` and "overwrite" `addListener()` and `addSubscriber()` methods to wrap added listeners with functions that would "unpack" the event and pass the new one (for `RenderGroupEvent` and `RenderSingleEvent` listeners only).

But that would require decorating the whole `EventDispatcher` just to handle this edge case, so I'm okay with it remaining like this. Unless there is an issue with new and old events being handled out of order, as is happening in this case.

@adamwojs giving this as a suggestion for consideration.